sky/cli.py

```diff
@@ -1034,6 +1036,13 @@ def cli():
     help=('[Experimental] Clone disk from an existing cluster to launch '
           'a new one. This is useful when the new cluster needs to have '
           'the same data on the boot disk as an existing cluster.'))
+@click.option(
+    '--fast',
+    is_flag=True,
+    default=False,
+    required=False,
+    help=('[Experimental] If the cluster is already up and available, skip'
```

Comment:
Looks like we also skip file_mount sync. Should we add that in this help str?

Reply:
I think I will actually just add the SYNC_FILE_MOUNTS stage back in, since it will be needed for managed jobs and other things that will use --fast.